api/src/main/java/org/pmiops/workbench/db/model/CohortAnnotationDefinition.java

```diff
@@ -78,6 +83,30 @@ public CohortAnnotationDefinition annotationType(AnnotationType annotationType)
         return this;
     }
 
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "cohortAnnotationDefinition", orphanRemoval = true, cascade = CascadeType.ALL)
```

Comment:
You may want to make this a SortedSet with OrderBy like here:

https://stackoverflow.com/questions/19871765/jpa-hibernate-sorted-collection-orderby-vs-sort

Reply:
Changed it to order by primary key.